platform/ios/src/MGLMapboxEvents.m

```diff
@@ -278,16 +286,24 @@ - (void)pauseMetricsCollection {
 }
 
 - (void)resumeMetricsCollection {
-    if (!self.paused || ![[self class] isEnabled]) {
+    if (!self.paused || ![[self class] isEnabled] || [self isLowPowerModeEnabled]) {
         return;
     }
-    
+
     self.paused = NO;
     self.data = [[MGLMapboxEventsData alloc] init];
     
     [self.locationManager startUpdatingLocation];
 }
 
+- (BOOL)isLowPowerModeEnabled {
```

Comment:
This can be a class method. In fact, it may make sense to consolidate this check with `+isEnabled`.